### PR TITLE
Mobile menu implementation with javascript

### PR DESCRIPTION
**Pull request by ghost**
This pull request is an attempt to implement an interactive menu in the mobile version of the portfolio using javascript manipulation.

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @iKuartz ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😢

## Invalid Code Review Request 🔴 

You have submitted a project, that does not have a pull request description.

Having a meaningful PR title and description will make it easy for anyone (reviewer or recruiter) to understand what was worked on in the milestone the PR is being created for.

Here is a [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects) to learn how to create a professional repository. 😁

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

Invalid Code Review Request does not count into the code reviews limit.

File: menu.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+const closingButton = document.getElementById('close');
+
+function toggleMenu() {
+  const nav = document.getElementById('nav');
+  nav.classList.toggle('menu-open');
+}
+
+buttonMobile.addEventListener('click', toggleMenu);
+closingButton.addEventListener('click', toggleMenu);
+document.querySelectorAll('.menu_links').forEach((item) => { const nav = document.getElementById('nav'); item.addEventListener('click', () => { nav.classList.remove('menu-open'); }); });
```
Comment (DammyShittu) [inline menu.js:11]: [OPTIONAL]

- Awesome implementation of the mobile menu feature. Well done 👍 What do you think about keeping the codebase clean by separating the code on line 11into multiple lines? 

Here 👇  is a cleaner/more readable version of your code 😀

**Review by DammyShittu — APPROVED** [review]
Hi @iKuartz , 

# APPROVED 🟢 

_Outstanding implementation of the features in this milestone .✔️_

Kudos to you 💯 

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

## CONGRATULATIONS! 🎉 

# OPTIONAL SUGGESTIONS ♻️ 

Kindly check the comment below 👇 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

# Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: menu.js (hunk 1 above)
Comment (ghost) [reply menu.js:11]: Thank you for your excellent suggestion. I definitely will.

**Pull request merged**